config/envs/default.py

```diff
@@ -154,6 +158,18 @@ class DefaultConfig(object):
         FUND_STORE_API_HOST + CommonConfig.ROUND_ENDPOINT
     )
 
+    # Post-award frontend
+    POST_AWARD_FRONTEND_HOST = environ.get("POST_AWARD_FRONTEND_HOST", "")
+    POST_AWARD_FRONTEND_LOGIN_URL = POST_AWARD_FRONTEND_HOST + "/"
+
+    # Safe list of return applications
+    SAFE_RETURN_APPS = {
```

Comment:
Guess eventually we'll expand this dict with entries for say `pre-award-application` & `pre-award-assessment`? 

I like the way you've done this! +1 for the named tuples as-well.

Reply:
that's the idea yeah!